apps/web/src/app/(authenticated)/organization/actions.ts

```diff
@@ -62,28 +62,40 @@ async function deleteOrganization(
   return await handleUrqlRequest(urqlClientSdk().deleteOrganization(values));
 }
 
-export async function createAndSwitchOrganization(values: CreateOrganizationMutationVariables): Promise<boolean> {
+export async function createAndSwitchOrganization(values: CreateOrganizationMutationVariables): Promise<{
+  success: boolean;
+  message?: string;
+}> {
   try {
     const res = await createOrganization(values);
 
     if (!res.success) {
       logger.error(res.error);
-      return false;
+      return {
+        success: false,
+        message: res.error,
+      };
```

Comment:
```suggestion
      return res;
```
I think it is the same signature with UrqlError. Also the logger.error is not needed, we log the error inside the urql. Applicable in one more place in the same file.

Reply:
Removed the logs.
UrqlErrorResult seems to be a bit different, but in my attempt to use it, it expects <TError> argument which I couldn't exactly import into the actions.ts file.

Reply:
Fixed.

Reply:
still see the object instead of `return res` :P